index.js

```diff
@@ -8,18 +8,14 @@ const defaultOptions = {
   tryWindow: true,
   tryApi: true,
   tryJsIpfs: false,
-  defaultApiOpts: {
-    host: '127.0.0.1',
-    port: '5001',
-    protocol: 'http'
-  }
+  defaultApiAddress: '/ip4/127.0.0.1/tcp/5001'
 }
 
 module.exports = (opts = {}) => {
   opts = Object.assign({}, defaultOptions, opts)
 
   const defaultState = {
-    apiOpts: opts.defaultApiOpts,
+    apiAddress: opts.defaultApiAddress,
```

Comment:
let's init this to the localStorage value if available

Reply:
Done"